extension/chrome/settings/setup.htm

```diff
@@ -147,7 +147,7 @@ <h1>Set Up FlowCrypt</h1>
 
       <div class="line left">
         <label>
-          Encryption key type:
+          Encryption key family:
```

Comment:
Here the "key type" referred to algorithms, not openpgp vs x509

Reply:
ok